File: source/examples/Relationships/ImplicitInverse/ImplicitInverse.js
{
  name: "Person",
  properties: {
    name: "string",
    birthdate: "date",
    dogs: "Dog[]"
  }
}

{
  name: "Dog",
  properties: {
    name: "string",
    age: "int",
    breed: "string?",
    owners: {
      type: 'linkingObjects',
      objectType: 'Person',
      property: 'dogs'
    }
  }
};

File: source/examples/Relationships/ImplicitInverse/ImplicitInverse.swift
class Person: Object {
    @objc dynamic var name: String = ""
    @objc dynamic var birthdate: Date = Date(timeIntervalSince1970: 1)
    let dogs = List<Dog>()
}

class Dog: Object {
    @objc dynamic var name: String = ""
    @objc dynamic var age: Int = 0
    @objc dynamic var breed: String? = nil
    let owners = LinkingObjects(fromType: Person.self, property: "dogs")
}
